### PR TITLE
[5.1] one WebApp for one DIRAC Setup

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

FIX: Removed Setup from SystemAdministrator
CHANGE: one WebApp for one DIRAC Setup

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
Review, please.

**Comment by aldbr** [comment]
What about the `DIRACSetup` in `JobMonitor`:

https://github.com/DIRACGrid/WebAppDIRAC/blob/integration/src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js#L142-L144

https://github.com/DIRACGrid/WebAppDIRAC/blob/integration/src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js#L454-L459

**Comment by fstagni** [comment]
> What about the `DIRACSetup` in `JobMonitor`:
> 
> https://github.com/DIRACGrid/WebAppDIRAC/blob/integration/src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js#L142-L144
> 
> https://github.com/DIRACGrid/WebAppDIRAC/blob/integration/src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js#L454-L459

Those have nothing to do with this PR, but they had to be removed anyway as a consequence of https://github.com/DIRACGrid/DIRAC/pull/6164 ! Thanks, done.

File: src/WebAppDIRAC/WebApp/handler/SystemAdministrationHandler.py [hunk 1]
```diff
@@ -22,13 +22,13 @@ def web_getSysInfo(self):
             return {"success": "false", "error": result.get("Message", "No system information found")}
 
         # Add the information about the extensions' versions, if available, to display along with the DIRAC version
-        for i in range(len(callback)):
-            callback[i]["Host"] = callback[i]["HostName"]
-            callback[i]["Timestamp"] = str(callback[i].get("Timestamp", "unknown"))
+        for i, cb in enumerate(callback):
+            callback[i]["Host"] = cb["HostName"]
+            callback[i]["Timestamp"] = str(cb.get("Timestamp", "unknown"))
```
Comment (aldbr) [inline src/WebAppDIRAC/WebApp/handler/SystemAdministrationHandler.py]:
```suggestion
            cb["Host"] = cb["HostName"]
            cb["Timestamp"] = str(cb.get("Timestamp", "unknown"))
```

**Review by aldbr — APPROVED** [review]

**Pull request merged**